WhaleSpotting/react-app/src/apiModels/CreateSightingApiModel.ts

```diff
@@ -17,6 +17,44 @@ export enum Species {
     Unknown,
 }
 
+export const WhaleImageDictionary = {
```

Comment:
Can we add these 2 consts and the 2 enums to there own files (maybe an ApiLookups file and an ApiEnums file)

Reply:
Addressed.